tests/Benchmarks/README.md

```diff
@@ -0,0 +1,32 @@
+# CoreFxLab Benchmarks
+
+This project contains performance benchmarks.
+
+## Run the Performance Tests
+
+**Pre-requisite:** The dotnet cli is available at the root in the dotnetcli directory. On a clean repo, build.cmd at the root installs the latest available dotnet.exe.
+
+**Pre-requisite:** To use dotnet cli from the root directory remember to set `DOTNET_MULTILEVEL_LOOKUP` environment variable to `0`!
+
+    $env:DOTNET_MULTILEVEL_LOOKUP=0
+
+1. Navigate to the benchmarks directory (corefxlab\tests\Benchmarks\)
+
+2. Restore this project
+   
+   (`..\..\dotnetcli\dotnet.exe restore`)
+
+3a. Run the benchmarks in Release, choose one of the benchmarks when prompted
+   
+   (`..\..\dotnetcli\dotnet.exe run -c Release`)
+   
+3b. To run specific tests only, pass in the filter to the harness:
+   
+   (`..\..\dotnetcli\dotnet.exe run -c Release -- --filter namespace*`)
+   (`..\..\dotnetcli\dotnet.exe run -c Release -- --filter *typeName*`)
+   (`..\..\dotnetcli\dotnet.exe run -c Release -- --filter *.methodName`)
+   (`..\..\dotnetcli\dotnet.exe run -c Release -- --filter namespace.typeName.methodName`)
```

Comment:
nit: The steps are not getting formatted correctly when we view the README within the browser.

- The numbered list is not aligned correctly.
- The various filter commands all end up on the same line.

![image](https://user-images.githubusercontent.com/6527137/42789022-b7f91e82-8917-11e8-97ec-20b9d468fcf9.png)